RevenueCatUI/CustomerCenter/Mocks/MockCustomerCenterPurchases.swift

```diff
@@ -118,6 +118,27 @@ final class MockCustomerCenterPurchases: @unchecked Sendable, CustomerCenterPurc
         return try restorePurchasesResult.get()
     }
 
+    var invalidateVirtualCurrenciesCacheCallCount = 0
+    func invalidateVirtualCurrenciesCache() {
+        invalidateVirtualCurrenciesCacheCallCount += 1
+    }
+
+    var virtualCurrenciesCallCount = 0
+    var virtualCurrenciesResult: Result<VirtualCurrencies, Error>?
+    func virtualCurrencies() async throws -> VirtualCurrencies {
+        virtualCurrenciesCallCount += 1
+        guard let virtualCurrenciesResult = virtualCurrenciesResult else {
+            return VirtualCurrenciesFixtures.noVirtualCurrencies
+        }
+
+        switch virtualCurrenciesResult {
+        case .success(let virtualCurrencies):
+            return virtualCurrencies
+        case .failure(let error):
+            throw error
+        }
```

Comment:
This all can be replaced by this one liner, but FFTI
```suggestion
        return try virtualCurrenciesResult?.get() ?? VirtualCurrenciesFixtures.noVirtualCurrencies
```